packages/design-system/src/components/Table/TableCaption.jsx

```diff
@@ -0,0 +1,51 @@
+import Alert from '../Alert/Alert';
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableCaption = ({
+  children,
+  className,
+  scrollActive,
+  scrollCaption,
+  ...tableCaptionProps
+}) => {
+  const classes = classNames('ds-c-table__caption', className);
+
+  return (
+    <caption className={classes} {...tableCaptionProps}>
+      {children}
+      {scrollActive && scrollCaption}
+    </caption>
+  );
+};
+
+TableCaption.defaultProps = {
+  scrollCaption: (
+    <Alert className="ds-u-margin-y--1 ds-u-font-size--small ds-u-font-weight--normal">
```

Comment:
Regarding one of my previous comments: https://github.com/CMSgov/design-system/pull/706#issuecomment-646247189

Can we change the role on this alert from region to status? In my testing, that makes the keyboard message announce in IE11 and Safari.

Reply:
@davidakennedy I've changed role to 'status'. From reading up on role status, it mentions that this status is similar to aria-live="polite" & aria-relevant='additions', so I've removed those attributes too.